expan/core/experiment.py

```diff
@@ -265,10 +273,10 @@ def _is_valid_for_analysis(self, data, test):
 
     def _get_denominators(self, data, test, variant_name):
         if type(test.kpi) is not DerivedKPI:
-            return 1.0
+            return np.float64(1.0)
```

Comment:
why is this important?

Reply:
It might not be important actually!

In our internal software, we get errors like the following:

    TypeError: can't multiply sequence by non-int of type 'float'

This suggest a situation where there is confusion about whether to interpret `[3,4,5] * 2.0` as `[6,8,10]` or as `[3,4,5]*2 == [3,4,5,3,4,5]`. To avoid ambiguity, we should consider everything as a `numpy` type